pox/pox/ext/utils.py

```diff
@@ -0,0 +1,30 @@
+# Utility functions
+
+from topologies import topologies
+
+def build_topology(topo):

```

Comment:
What is this for?

Reply:
When you launch a controller, it can only take string arguments.  So we need a way to transform an argument like `--topo=jellyfish,arg1,arg2` to an actual Jellyfish topology object with those arguments given to its constructor. This is the utility function that does that